src/main/java/sg/ncl/MainController.java

```diff
@@ -535,7 +541,7 @@ public String loginSubmit(
         }
 
         String jwtTokenString = response.getBody().toString();
-        log.info("token string {}", jwtTokenString);
+        // log.info("token string {}", jwtTokenString);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)